docker/jenkins/run_mcom_tests.sh

```diff
@@ -1,11 +1,4 @@
 DOCKER_REPOSITORY=${DOCKER_REPOSITORY:-mozorg/mcom-tests}
 
-SAUCE_CREDENTIALS_PATH=`mktemp`
-cat << EOF > $SAUCE_CREDENTIALS_PATH
-username: $SAUCELABS_USERNAME
-password: $SAUCELABS_PASSWORD
-api-key: $SAUCELABS_API_KEY
-EOF
-
 docker pull ${DOCKER_REPOSITORY}
-docker run -v $SAUCE_CREDENTIALS_PATH:/tmp/creds -v `pwd`/results/:/app/results -e BASE_URL=${BASE_URL} -e BUILD_TAG=${BUILD_TAG} ${DOCKER_REPOSITORY}
+docker run -v `pwd`/results/:/app/results -e BASE_URL=${BASE_URL} -e BUILD_TAG=${BUILD_TAG} -e SAUCE_LABS_USERNAME=${SAUCE_LABS_USERNAME} -e SAUCE_LABS_API_KEY=${SAUCE_LABS_API_KEY} ${DOCKER_REPOSITORY}
```

Comment:
Nice, catch. Updated! I intentionally left `${SAUCE_LABS_USERNAME}` and `${SAUCE_LABS_API_KEY}` b/c that's how jenkins is configured.